### ISSUE TITLE
[SparklineChart-SparklineChart basic]: Screen reader fails to announce 'Sparkline charts' and 'values' when user navigates table content.

### ISSUE THREAD

**Issue by AtishayMsft** [microsoft/fluentui#30854]
Test Environment:
OS Build: Windows 11
Version: 24H2 (OS Build 26058.1400)
Browser: Edge-dev
URL: [Fluent UI - Controls - React - SparklineChart (microsoft.com)](https://developer.microsoft.com/en-us/fluentui#/controls/web/sparklinechart)
Browser Version: 123.0.2420.10 (Official build) dev (64-bit)
Screen reader: NVDA 2023.3.4(2023.3.4.31265)

Pre-requisites:
Turn on NVDA using 'Ctrl+ alt+ N' keys.
Repro Steps: 
Open the above-mentioned [URL](https://developer.microsoft.com/en-us/fluentui#/controls/web/sparklinechart) in edge-dev browser.
'SparklineChart' page opens.
Navigate to 'SparklineChart' under 'SparklineChart basic' section using tab key.
Navigate to the data values using arrow key.
Observe the screen reader announcement.
Actual Result:
In a table containing 8 sparkline charts labeled from 'Row1' to 'Row 8', screen readers fails to announce information about the sparkline charts and their corresponding values.

When navigating the table with charts and values, the screen reader announces 'graphic' when reaching the charts along with their values.

Observation:
In browse mode, when navigating over a chart, it announces as 'graphic graphic Sparkline with label 19.64', and when navigating over the value, it announces as 'graphic 19.64'.

Expected Result:
When navigating over the chart in 'Row1', the screen reader should announce something like: 'Row 1 sparkline chart with label 19.64 graphic'.
When navigating on the value for 'Row 1', it should announce the value as '19.64 graphic'.

Note:
Same issue is repro with Narrator screen reader.

User Impact:
Users relying on screen readers may experience difficulty in understanding and interpreting the information presented in the sparkline charts. Inaccurate or incomplete screen reader announcements hinder their ability to access data effectively, impacting the overall accessibility and usability of the interface.

![Image](https://github.com/microsoft/fluentui/assets/98592573/aa332963-ec17-4775-92b2-1d7eb1d5f2b6)
https://github.com/microsoft/fluentui/assets/98592573/ce6606ae-17e4-47df-89d9-cc2f6bd6c600
![Image](https://github.com/microsoft/fluentui/assets/98592573/736ba0a8-08d5-444e-8ddc-33328eb88263)
